virtualmachine/vsphere/vm.go

```diff
@@ -1711,9 +1713,12 @@ func getVmInfo(vmMo mo.VirtualMachine, vmPath string) map[string]interface{} {
 		}
 		backing := disk.Backing.(types.BaseVirtualDeviceFileBackingInfo)
 		fileBackingInfo := backing.GetVirtualDeviceFileBackingInfo()
+
+		diskSizeGB := disk.CapacityInKB / (1024 * 1024)
```

Comment:
we should return the float value here, closer to accurate value.

Reply:
We are currently accepting Disk size in integer for CreateVMs and AddDisk, so if want to change response then we should change type for size in requests also. Should we do that?

Reply:
A float type should accept the integer values, if client/controller sends integer it will be accepted. Please verify once.